samples/unity-of-bugs/Assets/Editor/Builder.cs

```diff
@@ -57,6 +63,53 @@ public static void BuildIl2CPPPlayer(BuildTarget buildTarget)
     public static void BuildAndroidIl2CPPPlayer() => BuildIl2CPPPlayer(BuildTarget.Android);
     public static void BuildIOSPlayer() => BuildIl2CPPPlayer(BuildTarget.iOS);
 
+    private static void SetupSentryOptions(Dictionary<string, string> args)
+    {
+        Debug.LogFormat(LogType.Log, LogOption.NoStacktrace, null, "SetupSentryOptions: Invoking SentryOptions");
+
+        if (!EditorApplication.ExecuteMenuItem("Tools/Sentry"))
+        {
+            throw new Exception("SetupSentryOptions: Menu item Tools -> Sentry was not found.");
+        }
+
+        var sentryWindowType = AppDomain.CurrentDomain.GetAssemblies()
+            ?.FirstOrDefault(assembly => assembly.FullName.StartsWith("Sentry.Unity.Editor,"))
+            ?.GetTypes()?.FirstOrDefault(type => type.FullName == "Sentry.Unity.Editor.SentryWindow");
+        if (sentryWindowType is null)
+        {
+            foreach (var asm in AppDomain.CurrentDomain.GetAssemblies().Where(name => name.FullName.Contains("Sentry")))
+            {
+                foreach(var type in asm.GetTypes())
+                {
+                    Debug.LogFormat(LogType.Log, LogOption.NoStacktrace, null, "SetupSentryOptions: Asm {0} Type {1}", asm.FullName, type.FullName);
+                }
+            }
+            throw new EntryPointNotFoundException("SetupSentryOptions: Type SentryWindow not found");
+        }
+
+        var optionsWindow = EditorWindow.GetWindow(sentryWindowType);
+        var options = optionsWindow.GetType().GetProperty("Options").GetValue(optionsWindow);
+
+        if (options is null)
+        {
+            throw new EntryPointNotFoundException("SetupSentryOptions: Method SentryOptions not found");
+        }
+        Debug.LogFormat(LogType.Log, LogOption.NoStacktrace, null, "SetupSentryOptions: Found SentryOptions");
+
+        var dsn = args["sentryOptions.Dsn"];
+        if (dsn != null)
```

Comment:
Used pattern matching above, keep consistent?
```suggestion
        if (dsn is not null)
```

If Unity can't compile this, then `dsn is { }` might do the job for now